src/components/shared/Validations.tsx

```diff
@@ -13,7 +15,7 @@ export const Validations: React.FunctionComponent<IValidations> = ({
   required,
   validations: { deprecated, ...validations },
 }) => {
-  const validationCount = Object.keys(validations).length;
+  const validationCount = Object.keys(validations).filter(validation => validation !== 'format').length;
```

Comment:
Let's just skip `format` in validations entirely.

Reply:
I skip the `format` in lines 36-38. Is that what you had in mind?

Reply:
Ah, sorry. I wasn't clear enough.
I meant skipping them in `src/utils/getValidations.ts`

Reply:
https://github.com/stoplightio/json-schema-viewer/blob/f9d11d6a54de347ff283a9bd9f5ea5e729dd0441/src/utils/getValidations.ts#L7